src/index.js

```diff
@@ -9,14 +9,12 @@ import App from './App';
 import './index.css';
 import reducer from './reducers';
 
-const store = createStore(
-  reducer,
-  compose(
-    applyMiddleware(thunk),
-    window.__REDUX_DEVTOOLS_EXTENSION__ &&
-      window.__REDUX_DEVTOOLS_EXTENSION__(),
-  ),
-);
+const preloadedState = window.__REDUX_DEVTOOLS_EXTENSION__ &&
```

Comment:
The correct parameter name is 'enhancer' and not 'preloadedState'. Please check the docs and correct it accordingly.